README.md

```diff
@@ -58,6 +58,20 @@ $ curl http://0.0.0.0:8080/record/23
 ## Next.js app
 When you run for the first time, you will need to copy the template .env.example file and add missed variables:
 
-   ```sh
-   $ cp .env.example .env
-   ```
\ No newline at end of file
+```sh
+$ cp .env.example .env
+```
+
+I've added `docker-compose-dev` - use this file if you need to make changes to the code. Next.js will compile into a dev build automatically after any changes are made to the JS (Type Script) code. This build is convenient for development, but it's about 30-40% slower than the "production" build. Therefore, if you want to get maximum performance, use:
+```
+$ docker-compose build 
+$ docker-compose up
+```
+If you want the ability to change the code and see changes immediately, use:
+
+```
+$  docker-compose -f docker-compose-dev.yml  build
+$  docker-compose -f docker-compose-dev.yml  up
+```
```

Comment:
We don't need two separate docker compose flows - we're not using it for any deployments. 

Ideally we shouldn't need two Dockerfiles either - if the only difference is `yarn start` vs `yarn dev`, this can be controlled by the run command in docker compose, and in Digital Ocean.